src/hypertable_cache.c

```diff
@@ -14,10 +14,10 @@ static void *hypertable_cache_create_entry(Cache *cache, CacheQuery *query);
 
 typedef struct HypertableCacheQuery
 {
-	CacheQuery	q;
 	Oid			relid;
 	const char *schema;
 	const char *table;
+	CacheQuery	q;
```

Comment:
VS doesn't like something that contains 0-sized arrays to be anything but the last item in a struct.

Reply:
PostgreSQL has a define `FLEXIBLE_ARRAY_MEMBER` to deal with this I think. Look in `pg_config.h.win32` or search the headers to see if this helps.

Reply:
Thanks, updated.